src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,95 @@
+package duke.task;
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidTaskTypeException;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+
+public class TaskManager {
+    private Task[] tasks = new Task[100];
```

Comment:
Perhaps can consider the use of variable length array like ArrayLists. There could be some bugs that may happen if the user decide to add more than 100 tasks.